2017-02-08 09:21:40,389 httprequest.py[line:50] ERROR 发送get请求: http://114.55.40.35:806/V1/api/Sys/GetUKeySecAndUtcDiff?clienUtcDiff=1476352997   服务器返回:  404
 error info: {"message":"No HTTP resource was found that matches the request URI 'http://114.55.40.35:806/V1/api/Sys/GetUKeySecAndUtcDiff?clienUtcDiff=1476352997'."} 
2017-02-08 09:21:40,389 dataparse.py[line:33] ERROR 'success'
2017-02-08 09:22:00,978 httprequest.py[line:48] INFO 发送get请求: http://114.55.40.35:806/V1/api/Sys/GetUKeySecAndUtcDiff?clientUtcDiff=1476352997   服务器返回:  200
